packages/opc-header/README.md

```diff
@@ -0,0 +1,48 @@
+# This is opc-header 👋
+
+Web component based on lit-element, it can be used as a other element of HTML example <opc-header> </opc-header>
+
+https://docs.google.com/document/d/1PoxpPsret2njugkAG_YBmlS3ZNkDd0MTY-JZKDbesrQ/edit#heading=h.w85tasqfwpfc
+
+## Install
+
+```sh
+npm i  @1-Platform/opc-header 
+```
+
+## Usage
+
```

Comment:
We might need examples to go with this. For example
```suggestion
<opc-header header-name="Outages"></opc-header>
```
And adding images would be great too